examples/auto-test/job_manage.py

```diff
@@ -0,0 +1,124 @@
+from hdfs import Client
+import urllib3
+import json
+
+
+class JobManager(object):
+    def __init__(self, config, expiration=30000):
+        def get_token(username, password, expiration):
+            ###
+            # input_type: str, str, int
+            # input: the username of PAI, the password of PAI and the expiration time of the token
+            # output_type: str
+            # output: token
+            # Get the token from rest server API
+            ###
+            token_ready = False
+            while not token_ready:
```

Comment:
Please sleep for a while in the loop to avoid causing much traffic to rest-server.

Reply:
done